v2/slogger/logger.go

```diff
@@ -325,12 +324,12 @@ func (self *StackError) Error() string {
 
 func stripDirectories(filepath string, toKeep int) string {
 	var idxCutoff int
-	if idxCutoff = strings.LastIndex(filepath, string(os.PathSeparator)); idxCutoff == -1 {
+	if idxCutoff = strings.LastIndex(filepath, string('/')); idxCutoff == -1 {
```

Comment:
Yes.  Turns out that the go runtime uses forward slashes in stack traces on Windows.